src/App.js

```diff
@@ -16,18 +16,31 @@ const App = () => {
   const createNewCat = (newCat) => {
     console.log(newCat)
   }
+  const updateCat = (editCat, id) => {
+    console.log(editCat, id)
+  }
+  const deleteCat = (id) => {
+    setCats(cats.filter(cat => cat.id !== id));
```

Comment:
I don't think you want to call setCats here. That will update your cats in state to be just one cat.